Readme.md

```diff
@@ -49,6 +49,7 @@ extension=/opt/bref-extra/amqp.so
 | Memcache | `${bref:extra.memcached-php-74}` | `extension=/opt/bref-extra/memcache.so` |
 | Memcached | `${bref:extra.memcached-php-74}` | `extension=/opt/bref-extra/memcached.so` |
 | Xdebug | `${bref:extra.xdebug-php-74}` | `zend_extension=/opt/bref-extra/xdebug.so` |
+| PostgreSQL | `${bref:extra.pgsql-php-74}` | `extension=/opt/bref-extra/pgsql.so` |
```

Comment:
Could you move this line above Xdebug to keep them in alphabetical order?

Reply:
Sure. I will fix it.